main/src/cgeo/geocaching/utils/SystemInformation.java

```diff
@@ -219,7 +220,7 @@ private static void appendConnectors(@NonNull final StringBuilder body) {
                 if (connector instanceof ILogin) {
                     final ILogin login = (ILogin) connector;
                     connectors.append(": ").append(login.isLoggedIn() ? "Logged in" : "Not logged in")
-                            .append(" (").append(login.getLoginStatusString()).append(')');
+                            .append(" (localized message: ").append(login.getLoginStatusString() /* unfortunately localized but an English string would require large refactoring */).append(')');
```

Comment:
I think its not necessary here to prepend "Localized message" as it is easy to see